### PR TITLE
Add plot_ci function and test. And rename functions

### PR DESCRIPTION
**Pull request by zackt113**
I have renamed the plot functions everywhere.  Please help me to check if there is anything I missed. Much appreciated!

Zack

### CODE REVIEW

**Comment by voremargot** [comment]
Looks good to me! I am happy to merge

**Comment by zackt113** [comment]
Hi, just added my function and test.  Let me know how's that looks. @gauthampughazhendhi @voremargot

File: R/plot_ci.R [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                      size = 3, 
+                      color = "purple",
+                      fill=" lightgrey") +
+    ggplot2::ggtitle(title)
+  
+  ggplot2::ggsave(filename = paste(title, ".png"),
+                  plot = annotated_plot,
+                  path = save_result_to)
+
+  return(annotated_plot)
```
Comment (gauthampughazhendhi) [inline R/plot_ci.R]:
```suggestion
annotated_plot
```
I think you don't need `return` here, but please make sure you test it after making the change.

**Review by gauthampughazhendhi — CHANGES_REQUESTED** [review]
Looks good to me! I have only one change

**Comment by zackt113** [comment]
Fiixed it! And no problem with running tests

File: tests/testthat/test-plot_ci.R [hunk 2]
```diff
@@ -0,0 +1,38 @@
+test_that("Check if error thrown for invalid title input", {
+  expect_error(plot_ci(c(1, 2, 3), 1000,
+                       n=100,
```
Comment (voremargot) [inline tests/testthat/test-plot_ci.R]: Fix formatting
```suggestion
                       n = 100, 
```

File: tests/testthat/test-plot_ci.R [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  expect_error(plot_ci(c(1, 2, 3), 1000,
+                       n=100, 
+                       ci_level = 0.95, 
+                       ci_random_seed = 123, 
+                       title = 123))
+})
+
+test_that("Check if error thrown for invalid x_axis input", {
+  expect_error(plot_ci(c(1, 2, 3), 1000,
+                       n=100,
```
Comment (voremargot) [inline tests/testthat/test-plot_ci.R]:
```suggestion
                       n = 100, 
```

File: tests/testthat/test-plot_ci.R [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  expect_error(plot_ci(c(1, 2, 3), 1000,
+                       n=100, 
+                       ci_level = 0.95, 
+                       ci_random_seed = 123, 
+                       x_axis = 123))
+})
+
+test_that("Check if error thrown for invalid y_axis input", {
+  expect_error(plot_ci(c(1, 2, 3), 1000,
+                       n=100,
```
Comment (voremargot) [inline tests/testthat/test-plot_ci.R]:
```suggestion
                       n = 100, 
```

File: tests/testthat/test-plot_ci.R [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  expect_error(plot_ci(c(1, 2, 3), 1000,
+                       n=100, 
+                       ci_level = 0.95, 
+                       ci_random_seed = 123, 
+                       y_axis = 123))
+})
+
+test_that("Check if error thrown for invalid save_result_to input", {
+  expect_error(plot_ci(c(1, 2, 3), 1000,
+                       n=100,
```
Comment (voremargot) [inline tests/testthat/test-plot_ci.R]:
```suggestion
                       n = 100, 
```

File: R/plot_ci.R [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+#' @param x_axis A character vector for the name of the x axis
+#' @param y_axis A character vector for the name of the y axis
+#' @param save_result_to A character vector for the path to directory 
+#'                       from current directory to save plot
+#'
+#' @return ggplot object
+#' @export
+#'
+#' @examples
+#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000 ,n=100,
```
Comment (voremargot) [inline R/plot_ci.R]:
```suggestion
#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000, n = 100, 
```

File: R/plot_ci.R [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+#' @param save_result_to A character vector for the path to directory 
+#'                       from current directory to save plot
+#'
+#' @return ggplot object
+#' @export
+#'
+#' @examples
+#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000 ,n=100, 
+#'                   ci_level = 0.95, ci_random_seed = 123)
+#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000 ,n=100,
```
Comment (voremargot) [inline R/plot_ci.R]:
```suggestion
#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000, n = 100, 
```

File: README.Rmd [hunk 8]
```diff
@@ -48,16 +48,19 @@ devtools::install_github("UBC-MDS/strapr")
 
 This is a basic example which shows you how to solve a common problem:
 
-    library("strapr")
-    histogram_ci_plot([1, 1, 2, 3, 5, 10]， level=0.95）
+
+```{r example}
+# library("strapR")
+# plot_ci([1, 2, 3, 4, 5, 6, 7], 1000, n=100, ci_level=0.95, ci_random_seed=123)
```
Comment (voremargot) [inline README.Rmd]:
```suggestion
# plot_ci([1, 2, 3, 4, 5, 6, 7], 1000, n = 100, ci_level = 0.95, ci_random_seed = 123)
```

File: README.md [hunk 9]
```diff
@@ -61,8 +61,11 @@ devtools::install_github("UBC-MDS/strapr")
 
 This is a basic example which shows you how to solve a common problem:
 
-    library("strapr")
-    histogram_ci_plot([1, 1, 2, 3, 5, 10]， level=0.95）
+
+``` r
+# library("strapR")
+# plot_ci([1, 2, 3, 4, 5, 6, 7], 1000, n=100, ci_level=0.95, ci_random_seed=123)
```
Comment (voremargot) [inline README.md]:
```suggestion
# plot_ci([1, 2, 3, 4, 5, 6, 7], 1000, n = 100, ci_level = 0.95, ci_random_seed = 123)
```

**Review by voremargot — COMMENTED** [review]
I was just formatting my functions and found some very tiny formatting things in your code! I think I found them all but you may want to check before we merge!

**Pull request merged**